editor/editor/style.scss

```diff
@@ -1,20 +1,34 @@
+/**
+ * Editor Styles
+ */
+
+* {
+	box-sizing: border-box;
+}
```

Comment:
This is already added here https://github.com/WordPress/gutenberg/blob/add/polish/editor/assets/stylesheets/main.scss#L19 and it's relative to the editor only to avoid altering any other WordPress style (sidebar etc...)